src/components/RefinementList/__tests__/RefinementList-test.js

```diff
@@ -28,6 +28,7 @@ describe('RefinementList', () => {
         ...defaultProps,
         cssClasses: {
           root: 'root',
+          searchable: {},
```

Comment:
this is likely not necessary right?

Reply:
We use `searchable[cssClass]` in the component (and `undefined[cssClass]` throws) so I'm not sure there's other ways.

Reply:
You're right, this is a component test, not a widget test!